linera-web/pnpm-workspace.yaml

```diff
@@ -1,3 +1,2 @@
 packages:
-  - "signer"
```

Comment:
This one confuses me.  So `signer` is no longer in the workspace, i.e. it gets its own separate `node_modules` and `package-lock.json`?

Reply:
OK, now signer is back in the workspace but its lock files are not tracked.

Similarly removed the lock file for `linera-web` and added them to .gitignore.